### ISSUE TITLE
Google SSO stopped working after updating to v0.32.3

### ISSUE THREAD

**Issue by madhurk** [metabase/metabase#9738]
Hi all,

I updated from v0.31.2 to v0.32.3 and Google SSO stopped working.. wondering if anyone else is seeing this issue. We were setup to use https:// from the very beginning. What I'm seeing is that after Google authentication goes through, the application fails to redirect and stays on the login screen.

I reverted back to v0.31.2 and SSO login and redirect seems to be working fine again. 

Any help will be much appreciated. Thanks!

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by davvilla** [comment]
We're seeing the same issue. If you open up the developer console it shows a 401 error (unauthorized).

Creating a new google app and reseting the google app ID did not work. We ended up needing to revert.

**Comment by mazameli** [comment]
Thanks for reporting this, guys. We're experiencing it ourselves, too. Just wanted to let you know we're actively investigating this.

**Comment by camsaul** [comment]
Able to repro this and should have fix shortly.

The `POST /api/session/google_auth` endpoint is setting the `Set-Cookie` header in the response e.g.

```
metabase.SESSION=142c777b-93d4-48f4-99c6-441809e778b4;SameSite=Lax;HttpOnly;Max-Age=1209600
```

however the browser does not seem to be including it in subsequent requests. So I'm guessing the redirect from Google is breaking the `SameSite` directive. Should have fix shortly

**Comment by camsaul** [comment]
Fixed by #9747
